cmd/bbolt/README.md

```diff
@@ -0,0 +1,325 @@
+# Introduction to bbolt command line
+
+Bbolt is a tool for inspecting and examining bbolt databases by accessing the low level details of the bbolt database. <br>
+`bbolt` is a command line client for Bbolt databases. To install bbolt command-line please refer [here](https://github.com/etcd-io/bbolt#installing). 
+
+**Note**: [Etcd](https://github.com/etcd-io/etcd) uses bbolt database as its backend storage engine. In this document, we will be 
```

Comment:
I believe we normally write `etcd` lowercase rather than title case:

```suggestion
**Note**: [etcd](https://github.com/etcd-io/etcd) uses bbolt database as its backend storage engine. In this document, we will be 
```